src/xdocs/config_blocks.xml

```diff
@@ -1045,6 +1045,24 @@ allowedFuture.addCallback(result -> {
 &lt;module name=&quot;RightCurly&quot;/&gt;
         </source>
 
+        <p>Example: </p>
+        <source>
+public void test() {
+  if (foo) {
+    bar();
+  } else {     // OK
```

Comment:
Please swap examples so violation comes first and then `OK`. This makes the examples look like "This is how to resolve an issue".